### ISSUE TITLE
Multiple breakpoints on the same line are not removed correctly

### ISSUE THREAD

**Issue by zero-plusplus** [zero-plusplus/vscode-autohotkey-debug#48]
For example, set a breakpoint in the first line and the three comments below it and run the program. The actual breakpoint is modified on line 5.
```ahk
a = ""
; 1
; 2
; 3
b = ""
```

Then, if you rerun the script after removing the breakpoint on line 5, it will stop on line 5.

Let me explain the cause.
Three breakpoints are removed at once, which means that the event that sets the breakpoint is called three times.

However, I don't know why, but before the breakpoint registration process is complete, the next event is executed.

This causes already removed breakpoints to be re-set.

I don't know how to solve this problem.
But most people won't notice this problem. In fact, the problem has been around since 1.0.0, but I never noticed it myself.

I'll add it to the README as a known issue and treat it as a pending one.

**Comment by SAbboushi** [comment]
I'm impressed with how thorough you are in your testing

**Comment by zero-plusplus** [comment]
Thanks.
Bugs take up valuable user time. So I try to remove them as much as possible.

As long as I don't find any bugs I should be able to release it in a few days.
As long as there are no bugs...

**Comment by SAbboushi** [comment]
Ok - good luck!

**Comment by zero-plusplus** [comment]
It's finished.
You can use `1.6.0` as soon as the upload process is complete.

**Comment by SAbboushi** [comment]
Yay!  Congratulations!

**Comment by zero-plusplus** [comment]
The plan after this is to release `1.6.1` after fixing #21.

Then I focus on developing the main extension to support AutoHotkey. This process could take several months.

If you want to know how the project is progressing, look [here](https://github.com/users/zero-plusplus/projects/1).
I'm not used to this feature (Project) so it may be difficult to see it.
I'll do my best to learn to use it.

**Comment by SAbboushi** [comment]
> developing the main extension to support AutoHotkey

I'm curious: how is this different from what you have already been doing?

**Comment by zero-plusplus** [comment]
Originally the plan was to create a single extension with features such as syntax highlighting, IntelliSense and debugging to support AutoHotkey development.

However, due to the sheer size of the extension I decided to carve out just the debugging part. That's `vscode-autohotkey-debug`. This had the advantage of being easy to integrate with other people's extensions.

And now that `vscode-autohotkey-debug` was at a satisfactory level, I decided to go back to my original plan.
